README.md

```diff
@@ -30,13 +30,24 @@ If you encounter an issue related to these samples please [submit a new issue](h
 
 <br>
 
-## How to run .NET modules (Windows 10)
+## How to run the .NET module sample (Windows 10)
 ### Prerequisites
 1. Install latest [Git Client](https://https://git-scm.com/downloads).
 2. Install Visual Studio 2015 with Update 3.
 ### Quick Start
 1. `git clone https://github.com/Azure-Samples/iot-edge-samples.git`
-2. `cd dotnet`
-3. Open `\DotnetModuleSample\DotnetModuleSample.sln` with and build the solution.
+2. `cd visualstudio\2015\dotnet\DotnetModuleSample`
+3. Open the `DotnetModuleSample.sln` solution file.
 4. Build the DotnetModuleSample project (Ctrl + Shift + B).
 5. Click the `Start` button in the Visual Studio 2015 IDE or press the F5 key.
+
+## How to run .NET Standard v1.3 module sample (Windows 10)
+### Prerequisites
+1. Install latest [Git Client](https://https://git-scm.com/downloads).
+2. Install Visual Studio 2017.
```

Comment:
The latest VS 2017 will install .NET SDK 1.1.2 by default, we should update the configuration json file to use the correct version.